conduct/code-of-conduct.md

```diff
@@ -44,34 +45,34 @@ except as necessary to protect vulnerable people from intentional abuse.
+While this code of conduct applies to spaces associated with `<EVENT>`, if you are being harassed by a guest/participant of/at `<EVENT>` outside of our spaces, we still want to know about it. We will take all good-faith reports of harassment by `<EVENT>` participants seriously. This includes harassment outside our spaces and harassment that took place at any point in time. The `<EVENT>` organizing committee reserves the right to exclude people from `<EVENT>` based on their past behavior, including behavior outside `<EVENT>` spaces and behavior towards people who are not part of `<EVENT>`.
 
 In order to protect volunteers from abuse and burnout, we reserve the right to reject any report we believe to have been made in bad faith. Reports intended to silence legitimate criticism may be deleted without response.
 
-We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them, if we believe that doing so will increase the safety of `<EVENT>` members or the general public. We will not name harassment victims without their affirmative consent.
+We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them if we believe that doing so will increase the safety of `<EVENT>` participants or the general public. We will not name harassment victims without their affirmative consent.
 
 Conference staff will also provide support to victims, including, but not limited to:
 
 - Providing an escort
 - Contacting hotel/venue security or local law enforcement
-- Briefing key event staff to for response/victim assistance
+- Briefing key event staff for response/victim assistance
 - And otherwise assisting those experiencing harassment to ensure that they feel safe for the duration of the conference.
 
 ### Consequences
 
-If a participant engages in harassing behavior, the conference organizers may take any action they deem appropriate, ranging from issuance of a warning to the offending individual to expulsion from the conference with no refund, depending on the circumstances. `<EVENT>` reserves the right to exclude any participant found to be engaging in harassing behavior from participating in any further `<EVENT>` events, trainings or other activities.
+If a participant engages in behavior in violation of the CoC, the conference organizers may take any action they deem appropriate, ranging from the issuance of a warning to expulsion from the conference with no refund, depending on the circumstances. `<EVENT>` reserves the right to exclude any participant found to be engaging in harassing behavior from participating in any further `<EVENT>` events, trainings or other activities.
```

Comment:
"the issuance of a warning" -> "issuing a warning" ?

Reply:
Sure, ill fix.